src/super_gradients/training/datasets/detection_datasets/pascal_voc_detection.py

```diff
@@ -18,13 +22,16 @@
 class PascalVOCDetectionDataset(DetectionDataset):
     """Dataset for Pascal VOC object detection"""
 
-    def __init__(self, images_sub_directory: str, *args, **kwargs):
+    def __init__(self, images_sub_directory: str, download: bool = False, *args, **kwargs):
         """Dataset for Pascal VOC object detection
 
         :param images_sub_directory:    Sub directory of data_dir that includes images.
         """
+
         self.images_sub_directory = images_sub_directory
         self.img_and_target_path_list = None
+        if download:
+            PascalVOCDetectionDataset.download(kwargs.get("data_dir"))
```

Comment:
assert "data_dir" in kwargs

Reply:
added in https://github.com/Deci-AI/super-gradients/pull/332/commits/3cb87ded44227afef53deb1815d0ed8f2daf9f8f